src/Juvix/Compiler/Concrete/Language.hs

```diff
@@ -287,11 +292,49 @@ deriving stock instance (Ord (SyntaxDef 'Parsed))
 
 deriving stock instance (Ord (SyntaxDef 'Scoped))
 
+data ParsedFixityFields (s :: Stage) = ParsedFixityFields
+  { _fixityFieldsAssoc :: Maybe BinaryAssoc,
+    _fixityFieldsPrecSame :: Maybe (SymbolType s),
+    _fixityFieldsPrecBelow :: Maybe [SymbolType s],
+    _fixityFieldsPrecAbove :: Maybe [SymbolType s],
+    _fixityFieldsBraces :: Irrelevant (KeywordRef, KeywordRef)
+  }
```

Comment:
I'd prefer to include this data in ParsedFixityInfo directly.

Reply:
This encoding is better because it reflects in the AST that if we do not skip the fields, we at least have a reference to the brackets.
If we inlined everything in ParsedFixityInfo we would have
```
...
_fixityFieldsPrecAbove :: Maybe [SymbolType s],
_fixityFieldsBraces :: Irrelevant (Maybe (KeywordRef, KeywordRef))
...
```
So the types are not telling that `_fixityFieldsBraces == Nothing` implies `_fixityFieldsPrecAbove == Nothing`.

Also, I'd say that semantically it makes sense to have it separated